docs/download/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: page
+title: Download
+hide:
+  #- navigation
+  - toc
+---
+
+By installing you agree to the [Omnissa general terms](https://www.omnissa.com/general-terms/)
+
+
+| Horizon Server version                                                                                                         | Download Powercli module |
+|------------------------------------------------------------------------------------------------------------------------| --- |
+| 8.13/2406                    | [horizon4powercli_2406](releases/tag/2406/Assets/horizon4powercli_2406.zip) |
+
+### Supported Powershell versions
+1. Windows PowerShell 5.1
+2. PowerShell 7.x
+
+### Installation Prerequisites
+| OS Type                                      | .Net Version |                                    | Powershell Version |
+|----------------------------------------------|--------------|------------------------------------|---------------------|
+| Windows                                      | .NET Framework 4.7.2 or later <br> .NET Core 3.1|                   |Windows PowerShell 5.1<br>PowerShell 7.x 
+| Linux                                          |.NET Core 3.1 |                                    |PowerShell 7.x        |
+| macOS                                          |.NET Core 3.1 |                                    |PowerShell 7.x        |
+
+### Installation steps
+1. Download the horizon powercli zip file.
+2. Open PowerShell on your local machine.
+3. To view the folder paths to which you can extract the PowerCLI ZIP file, run the command: 
+          <br>  $env:PSModulePath
+4. Extract the contents of the PowerCLI ZIP file to one of the listed folders. 
+
+
+
+<br><br><hr>
+
+<p><strong>Important Information: <br></strong>Horizon PowerCLI are based on View APIs which are scheduled to be deprecated in future releases. For more information, visit the <a href="https://kb.omnissa.com/s/article/6000139?lang=en_US target="_blank">link</a></p>
```

Comment:
change to:

**Important Information:**

Horizon PowerCLI is based on View APIs which are scheduled to be deprecated in future releases. For more information, visit the [https://kb.omnissa.com/s/article/6000139?lang=en_US](https://kb.omnissa.com/s/article/6000139?lang=en_US)